### PR TITLE
Update backup/restore docs note about custom templates

### PR DESCRIPTION
**Pull request by rocodes**
Include our recommended path for custom template upgrades (rebuild instead of upgrade). Link to upstream dist-upgrade template-standalone-upgrade instructions.

### CODE REVIEW

File: docs/admin/reference/backup.rst [hunk 1]
```diff
@@ -152,12 +152,16 @@ measures it requires before any additional VMs are configured.
...
-  documentation on upgrading templates (for example:
-  `Fedora templates <https://www.qubes-os.org/doc/templates/fedora/#upgrading>`_,
+  If you are restoring your own customized VMs and templates, you will need to take
+  additional steps. You may decide to create new templates for your custom VMs and
+  provision them with the necessary applications/customizations (recommended), or
+  you may upgrade your existing templates following the upstream documentation
+  (`Fedora templates <https://www.qubes-os.org/doc/templates/fedora/#upgrading>`_,
   `Debian templates <https://www.qubes-os.org/doc/templates/debian/#upgrading>`_),
-  or contact Support.
+  then upgrade their package repositories to the Qubes 4.2 repoistories using
```
Comment (eloquence) [inline docs/admin/reference/backup.rst:161]: typo: `repoistories`

File: docs/admin/reference/backup.rst [hunk 2]
```diff
@@ -152,12 +152,16 @@ measures it requires before any additional VMs are configured.
...
+  ``qubes-dist-upgrade --template-standalone--upgrade`` (more information can be
```
Comment (eloquence) [inline docs/admin/reference/backup.rst:162]: It looks like this tool has to be installed first with `sudo qubes-dom0-update -y qubes-dist-upgrade` -- on my system it was not present, and the linked docs instruct the user to install it, but we should probably say so here as well.

Typo: `--template-standalone--upgrade` (double hyphen)

**Review by eloquence — CHANGES_REQUESTED** [review]
(A couple of nits)

**Comment by zenmonkeykstop** [comment]
Will apply fixes and update.

**Review by eloquence — APPROVED** [review]
LGTM.

**Pull request merged**